doc/doc.html

```diff
@@ -761,15 +760,218 @@ <h3>Shop with exclusive offers</h3>
+          <img src="../images/aa.jpg" />
+          <div class="card_content">
+            <h2>Alag Aasmaan</h2>
+            <p>by Anuv Jain</p>
+            <p>
+              "Alag Aasmaan” is the fifth studio single by Anuv Jain. It is a
+              beautiful song about a long-distance relationship where it’s hard
+              to bid farewell.
+            </p>
+          </div>
+          <div class="card_footer">
+            <div class="card_buttons">
+              <button>Listen</button>
+              <button>Lyrics</button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="code_container">
+        <iframe
+          src="https://carbon.now.sh/embed?bg=rgba%28249%2C237%2C212%2C1%29&t=twilight&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=16px&ph=20px&ln=false&fl=1&fm=IBM+Plex+Mono&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522card_main_container%2522%253E%250A%2520%2520%253Cimg%2520src%253D%2522..%252Fimages%252Faa.jpg%2522%2520%252F%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card_content%2522%253E%250A%2520%2520%2520%2520%253Ch2%253EAlag%2520Aasmaan%253C%252Fh2%253E%250A%2520%2520%2520%2520%253Cp%253Eby%2520Anuv%2520Jain%253C%252Fp%253E%250A%2520%2520%2520%2520%253Cp%253E%250A%2520%2520%2520%2520%2520%2520%2522Alag%2520Aasmaan%25E2%2580%259D%2520is%2520the%2520fifth%2520studio%2520single%2520by%2520Anuv%2520Jain.%2520It%2520is%2520a%2520beautiful%250A%2520%2520%2520%2520%2520%2520song%2520about%2520a%2520long-distance%2520relationship%2520where%2520it%25E2%2580%2599s%2520hard%2520to%2520bid%2520farewell.%250A%2520%2520%2520%2520%253C%252Fp%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card_footer%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522card_buttons%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cbutton%253EListen%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%253Cbutton%253ELyrics%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522card_icons%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522..%252Fimages%252Fheart.png%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%253Cbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522..%252Fimages%252Fsend.png%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fbutton%253E%250A%2520%2520%2520%2520%2520%2520%253Cbutton%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522..%252Fimages%252Fdots.png%2522%2520%252F%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E%250A"
+          style="border: 0; overflow: hidden"
```

Comment:
avoid inline stylying , provide a class for this and use across the snippets

Reply:
Okay.